src/common/io_util.cc

```diff
@@ -515,7 +515,8 @@ StatusOr<int> EvbufferRead(evbuffer *buf, evutil_socket_t fd, int howmuch, [[may
   if (int ret = evbuffer_read(buf, fd, howmuch); ret > 0) {
```

Comment:
Also we need to add it for the SSL part.

Reply:
ok, i will modify it later